packages/components/badge/src/badgeTypes/IconBadge.tsx

```diff
@@ -0,0 +1,59 @@
+import { BadgeProps, BadgeStyledProps } from '../badge.types';
+import {
+  BadgeIconStyled,
+  BadgeWrapperStyled,
+  IconStyled,
+} from '../badge.styles';
+import { deepMerge, useTheme } from '@douro-ui/react';
+import { ReactNode } from 'react';
+
+export const IconBadge = ({
+  position,
+  size,
+  children,
+  icon,
+  styled,
```

Comment:
I think the IconBadge should have a onClick prop to allow the user to use pass the function that hw want to use

Reply:
Let's check with the designers to see if this is the idea for the badge icon. For now, I don’t think we need to add this prop.

Reply:
The "x" already indicates a closing function, but if the user wants to use another function, he's allowed to pass it. onClick is optional, it will only be called if the onClick prop is passed in